products/datagrid/get-started.adoc

```diff
@@ -35,7 +35,7 @@ $ yum install java-1.8.0-openjdk-devel
 $ unzip jboss-datagrid-7.0.0-server.zip
 ----
 +
-4. Download link:http://www.jboss.org/download-manager/file/jboss-eap-7.0.0.zip[Red Hat JBoss EAP 7.0.0]
+4. Download link:http:#{site.download_manager_base_url}/download-manager/file/jboss-eap-7.0.0.zip[Red Hat JBoss EAP 7.0.0]
```

Comment:
the 'http:' prefix should not be on this URL. I've opened another PR to fix this.